library/src/main/java/com/bumptech/glide/module/AppliesOptions.java

```diff
@@ -1,11 +1,13 @@
 package com.bumptech.glide.module;
 
 import android.content.Context;
+import android.support.annotation.NonNull;
 import com.bumptech.glide.GlideBuilder;
 
 /**
  * An internal interface, to be removed when {@link GlideModule}s are removed.
  */
+@Deprecated
```

Comment:
These changes got buried at the end, but I want to make sure you see them. I've deprecated `AppliesOptions`, `RegistersComponents`, and `ManifestParser` since they're all linked to the old `GlideModule`. Was there a reason for not deprecating them in the first place?

Reply:
Nope no reason I don't think. It adds a couple of extra IDE warnings, but nowhere important.